src/dev/jest/babel_transform.js

```diff
@@ -23,4 +23,9 @@ module.exports = babelJest.createTransformer({
   presets: [
     require.resolve('@kbn/babel-preset/node_preset')
   ],
+  plugins: [
+    // enables jest to parse and execute dynamic import() calls
+    '@babel/plugin-syntax-dynamic-import',
+    'dynamic-import-node'
```

Comment:
I don't _hate_ with this, but I would really appreciate not needing to build EUI when running our tests. Ideally EUI can ship with a commonjs version in the future that applies this plugin to rewrite the dynamic imports with static ones, then we can use jest's `moduleNameMapper` config to rewrite `@elastic/eui(/.*)?` to `<rootDir>/node_modules/@elastic/eui/commonjs$1`.

Reply:
Sounds good to me!